{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "00b5f0b5_8331247f",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1001050
      },
      "writtenOn": "2024-04-26T09:18:51Z",
      "side": 1,
      "message": "It seems that the reason for the build failure is not related to this patch.",
      "revId": "3998f8bd821762dcb6f83f97d2726f95358e1df7",
      "serverId": "8f6f209b-db1a-4cbf-aa44-c8bc30e9bfda"
    }
  ]
}